lib/invoker/parsers/config.rb

```diff
@@ -38,6 +38,10 @@ def process(label)
       private
 
       def load_config
+        if is_global?
+          @filename = to_global_file
```

Comment:
I think thats because, if filename was given as a basic name, this method turns this into a full path. For example:

`invoker start foo` will become `invoker start ~/.invoker/foo.ini` 

There is a potential problem of not supporting this feature for Procfiles though.